web/cobrands/fixmystreet/offline.js

```diff
@@ -302,6 +302,14 @@ fixmystreet.offline = (function() {
         $('.moderate-display.segmented-control, .shadow-wrap, #update_form, #report-cta, .mysoc-footer, .nav-wrapper').hide();
         $('.js-back-to-report-list').attr('href', '/my/planned');
 
+        // On iOS we want to hide the photo fields on the offline inspector
```

Comment:
Pedantic, but I guess this should be in the following commit, as in this commit, offline support is just broken for everything because the changes in the following commit have not yet been made.